LongWarOfTheChosen/Config/XComAI.ini

```diff
@@ -1316,12 +1317,21 @@ MAX_SURPRISED_SCAMPER_PATH_LENGTH=7		; Scamper paths longer than this number of
 			Child[4]=TryPriorityHarborWave, \\
 			Child[5]=HuntEnemyWithCover) 
 	
-+Behaviors=(BehaviorName=AssassinReaction, NodeType=Selector, Child[0]=DoIfFlankedPrimeFallback, Child[1]=NeedsReload, Child[2]=TryCombatReadiness)
++Behaviors=(BehaviorName=AssassinReaction, NodeType=Selector, Child[0]=TrySlash, Child[1]=NeedsReload, Child[2]=MoveMeleeFANATICUnsafe, Child[3]=HuntEnemyUnsafe,  Child[4]=TryCombatReadiness)
 +Behaviors=(BehaviorName=SkipMoveIfLastAction, NodeType=Selector, Child[0]=IsLastActionPoint, Child[1]=SkipMove)
 
 +Behaviors=(BehaviorName=ChosenAssassinFirstAction, NodeType=Sequence, Child[0]=NotLastActionPoint, Child[1]=ChosenAssassinFirstActionSelector)
 +Behaviors=(BehaviorName=ChosenAssassinLastAction, NodeType=Sequence, Child[0]=IsLastActionPoint, Child[1]=ChosenAssassinLastActionSelector)
 
++Behaviors=(BehaviorName=TrySlash, NodeType=Sequence, Child[0]=IsAbilityAvailable-Slash_LW, Child[1]=SetTargetStack-Slash_LW, Child[2]=GenericSelectBestTarget, Child[3]=HasValidTarget-Slash_LW,  Child[4]=SelectAbility-Slash_LW)
+
++Behaviors=(BehaviorName=IsAbilityAvailable-Slash_LW, NodeType=Condition)
++Behaviors=(BehaviorName=IsAbilityReady-Slash_LW, NodeType=Condition)
++Behaviors=(BehaviorName=SelectAbility-Slash_LW, NodeType=Action)
++Behaviors=(BehaviorName=HasValidTarget-Slash_LW, NodeType=Condition)
++Behaviors=(BehaviorName=SetTargetStack-Slash_LW, NodeType=Action)
```

Comment:
Are these correct? Wondering whether you meant to have `AssassinSlash_LW` here instead.